Oppfordre til å ikke bruke placeholders i skjema-elementer

KenAJoh opened the pull request:
Closes #695 
Ønskes feedback på bruk av språk. Oppfordre/anbefaler/ønsker osv

KenAJoh committed:
Placeholder oppfordring for Input, Fnrinput, skjemagruppe og textarea

Forklaring om bruk av Placeholders lagt i skjema-gruppe sin accessibility.md og div skjemaelementer som kan ta i bruk placeholder linker til denne forklaringen.

KenAJoh committed:
Bedre forklaring av hvor lenke fører til

KenAJoh committed:
Merge branch 'master' into uu/advare-mot-placeholders

KenAJoh committed:
Utbedret språk

Lillebo reviewed (changes requested):
Veldig bra 👍 Skjønner hva du har tenkt med å la hoved-teksten bo under SkjemaGruppe, men tror kanskje ikke det er helt riktig sted for den å bo. SkjemaGruppe er en ganske spesifikk `fieldset`-komponent, og ikke en overordnet wrapper/konteiner for alle skjema-relaterte ting.

Hva tenker du om å la denne teksten få sin egen side under [Tilgjengelighet](https://design.nav.no/accessibility) - litt på samme måte som vi gjør med alt-tekster?

KenAJoh committed:
Laget egen accessibility-side for Placeholders

- Alle komponenter som kan bruke placeholder-attributten har nå en link til denne siden.
